Assets/Scripts/P2PNetcodeSample/Networking/EOSTransport.cs

```diff
@@ -561,6 +569,44 @@ private ulong GetClientId(ProductUserId userId)
                 return UserIdToClientId[userId];
             }
         }
+
+        /// <summary>
+        /// Check if the specified ClientID exists
+        /// </summary>
+        /// <param name="clientId">The ClientId to search for</param>
+        /// <returns></returns>
+        public bool ClientIDExist(ulong clientId) 
```

Comment:
In a similar way, this clientID mentioned would only work with the transportIDs that are internal to this class, so making this public would imply that the client IDS are the same as the rest of the client IDs in the NetworkManager.